keva/src/main/java/com/jinyframework/keva/storage/HashMapStorage.java

```diff
@@ -0,0 +1,30 @@
+package com.jinyframework.keva.storage;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+@Slf4j
+public class HashMapStorage<K, V> {
+    private final ConcurrentMap<K, V> hashMap = new ConcurrentHashMap<>();
+
+    public byte put(K key, V val) {
```

Comment:
Why not `byte` => `bool`

Reply:
I wanted to return 1/0 like Redis instead of true/false string